scripts/pre-bootstrap

```diff
@@ -0,0 +1 @@
+echo Please follow the instructions at https://github.com/nhsuk/dev-base-machine to set up a development environment
```

Comment:
Can another line be added to this file with the content:
`open https://github.com/nhsuk/dev-base-machine`
It will open the link in the default browser. Makes it a little easier for people to get to where they need to go.